### ISSUE TITLE
Update testnets page with single testnet

### ISSUE THREAD

**Issue by IstoraMandiri** [ethereumclassic/ethereumclassic.github.io#1404]
re #1402

we should support a different format on testnets page to deal with multiple faucets

**Comment by chris-mercer** [comment]
Yes it would be great to update this page to reflect one testnet moving forward.

It appears the https://github.com/eth-classic/mordor maintainers are unresponsive and the services offered there are offline and have been offline for quite a while. I believe this repo was managed by @q9f @soc1c, the contributor who launched the Mordor testnet at the ETC Summit in 2019.
+ Back then we had a PoW testnet named Morden (deprecated in 2019).
+ Also we had a PoA testnet named Kotti (deprecated in 2022).
+ The PoW Mordor is the testnet of choice and I think for most contributors, it makes sense to focus on one strong, well supported testnet for the network protocol developers as well as the application layer developers


Some issue to note for constructive work moving forward to any contributor that is reading and would like to lend a helping hand to the network. Also this may lay the framework for the needs of this page on the website.

1.) In 2022 we have seen an explosion of activity in the mining ecosystem. We should really clean up the documentation for this user base. These are the primary users of the client.
+ Guide to setting up a mining client with various hardware (ASIC and GPU guides) and services (Solo mining, Pooled mining).
+ Links to mining metric dashboards like DAG growths, profitability estimators per hardware, mining pool hashrate distribution (with a note that distributing the hashrate to multiple pools is good security practice)
+ Consider adding links with referral codes to ETC hardware (for raising funds for website maintenance)
+ Video carousel of mining specific videos from the active ETC mining community. Example: ASIC manufactures and GPU mining channels like RPM/Rabid Mining/many others. This will create a hub for miners to check and help foster a robust and active security providing community.
+ Links to the discord mining channel. These are the early users of the network capturing the emission and selling to cover production. Let's be inclusive and make sure their needs are addressed on the website.

2.) With the increase in security on the network. It now makes sense to deploy applications to ETC. We are seeing the early signs of application layer developers involvement on the network. Given we are an EVM chain, there is an enormous amount of development talent in the world. We need to be prepared for the moment all of their eyes pivot and look at ETC as their PoW EVM of choice. This is inbound within 1-2 years tops, so we need to get organized here.
+ Remove all the stale development guides from the 2019. The network and EVM tooling is much different these days. Example, Truffle is deprecated. Hardhat is losing ground to Foundry as the development environment of choice.
+ Guides for Mordor mining for testnet funds.
+ Build out https://github.com/ethereumclassic/mordortestnet repo so we have testnet tools to onboard developers
+ Make sure a new developer can get to deploying code with a few clicks. Active faucets. RPC endpoint links. Wallet integration links: metamask, wallet connect are the big ones.

3.) Network users are also increasing. We should make sure its easy for new users to obtain testnet funds to test simple things like sending a transaction.
+ ELI5 Guides for onboarding new users via Mordor testnet funds. Teach them to grab funds from a faucet and interact with the network

@DonaldMcIntyre you've been writing great guides. This would be a perfect place to highlight guides you've already written and potentially ideas for new guides moving forward.

Much of our content on the website is dated. We really should do a comb through and prune dated/obsolete material. The space in 2023 is content rich with video tutorials. We should feature as much of this new content as possible. It would be amazing if this page could be integrated into the the "ETC Contribute" app for the mining community.

Just a brain dump of what I see after looking at it last week. I'll do my best to submit PRs when i see an opportunity to clean up old content and add new relevant content in its place.

**Comment by q9f** [comment]
> It appears the https://github.com/eth-classic/mordor maintainers are unresponsive

I responded 3 days ago, just let me know what you need.

https://github.com/eth-classic/mordor/issues/18#issuecomment-1829768707
